basic-features.md

```diff
@@ -10,192 +10,197 @@ aliases: ['/docs-cn/dev/basic-features/']
-| [`caching_sha2_password` 认证](/system-variables.md#default_authentication_plugin)    |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |  N  |  N  |
-| [`tidb_sm3_password` 认证](/system-variables.md#default_authentication_plugin)        |  Y  |  Y  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |
-| [`tidb_auth_token` 认证](/system-variables.md#default_authentication_plugin) |  Y  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |
-| [与 MySQL 兼容的 `GRANT` 权限管理](/privilege-management.md)      |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
-| [动态权限](/privilege-management.md#动态权限)                      |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |  N  |
-| [安全增强模式](/system-variables.md#tidb_enable_enhanced_security) |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |  N  |
-| [日志脱敏](/log-redaction.md)                                    |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |
+| 安全                                                            | 6.5 | 6.4 | 6.3 | 6.2 | 6.1 | 6.0 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+| ------------------------------------------------------------------ | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| [传输层加密 (TLS)](/enable-tls-between-clients-and-servers.md)    |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
+| [静态加密 (TDE)](/encryption-at-rest.md)                          |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
+| [基于角色的访问控制 (RBAC)](/role-based-access-control.md)        |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
+| [证书鉴权](/certificate-authentication.md)                        |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
+| [`caching_sha2_password` 认证](/system-variables.md#default_authentication_plugin)    |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |  N  |  N  |
+| [`tidb_sm3_password` 认证](/system-variables.md#default_authentication_plugin)        |  Y  |  Y  |  Y  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |
+| [`tidb_auth_token` 认证](/system-variables.md#default_authentication_plugin) |  Y  |  Y  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |
+| [密码管理](/password-management.md) |  Y  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |  N  |
+| [与 MySQL 兼容的 `GRANT` 权限管理](/privilege-management.md)      |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |
+| [动态权限](/privilege-management.md#动态权限)                      |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |  N  |
+| [安全增强模式](/system-variables.md#tidb_enable_enhanced_security) |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |  N  |
+| [日志脱敏](/log-redaction.md)                                    |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  Y  |  N  |
 
 ## 数据导入和导出
```

Comment:
@easonn7 Do you want to add bi-directional replication here?

Reply:
@IANTHEREAL @easonn7 PTAL

Reply:
added in https://github.com/pingcap/docs-cn/pull/12232/commits/de947d1865e25e5f10ab7c6c4d0428aab7f976e7